src/combobox/index.ts

```diff
@@ -467,44 +487,88 @@ let exportCombobox;
     }
 
     private _actionHighlightFirstMenuItem() {
-      this._setHighlightAndActiveDescendantMenu(this._firstItemEl);
+      let firstItem = this._firstItemEl;
+      let itemIsDisabled = false;
+      for (let i = 0; i < this.items.length; i++) {
```

Comment:
You inputted text, and all matched items are disabled, then JS error occurs when pressing "Home" or "End".